libcontainer/configs/cgroup.go

```diff
@@ -57,6 +57,11 @@ type Cgroup struct {
 	// MEM to use
 	CpusetMems string `json:"cpuset_mems"`
 
+	// Process limit; set to `0' to disable limit.
+	// While technically `0' is a valid PID limit, it does not make sense in the
+	// context of a container -- it is identical to a limit of `1'.
```

Comment:
I don't think make 0 being a valid limit is necessary, since a cgroup without any processes is meaningless, and we agree that there would always be at least 1 process in container, so set `pids.max=1` would be enough to prevent any fork() or clone() in container.

And take the default value of the type (such as 0 of int, null of mapping etc..) as the invalid value is always what we do, except some special types such as bool (like cgroup.OomKillDisable). So I prefer take 0 as the invalid value, and change specs instead. WDYT?